connectors/tests/test_es_index.py

```diff
@@ -182,6 +184,20 @@ async def test_update_with_concurrency_control(mock_responses):
     await index.close()
 
 
+@pytest.mark.asyncio
+async def test_update_by_script():
+    doc_id = "1"
+    script = {"source": ""}
+    index = ESIndex(index_name, config)
+    index.client = Mock()
+    index.client.update = AsyncMock()
+
+    await index.update_by_script(doc_id, script)
+    index.client.update.assert_awaited_once_with(
+        index=index_name, id=doc_id, script=script
+    )
+
```

Comment:
(Possibly ignorant) question: I see that this test is only checking that the `self.client.update` was called, so you've set the arg `script = {"source": ""}`.
I'm guessing that it isn't possible to test if an actual script would work, right?

Reply:
>I'm guessing that it isn't possible to test if an actual script would work, right?

Could you elaborate? The test is just testing the correct params are passed to `update` function.

`if an actual script would work` sounds like it requires a real ES instance, which I think it's out of scope of unit test.